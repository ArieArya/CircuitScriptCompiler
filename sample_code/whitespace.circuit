wire       some_wire_1 = and(0, 1);
wire some_wire_2  = or(0,  1);
wire           some_wire_3    = and (1,  1)   ;

print  (  some_wire_3);