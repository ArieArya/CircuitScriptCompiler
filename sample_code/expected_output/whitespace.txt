Lexing file: whitespace.circuit
Tokens:
Token.OPERATOR (Lexeme = '==')
Token.OPERATOR (Lexeme = '=')
Token.OPERATOR (Lexeme = '=')
Token.OPERATOR (Lexeme = '==')
Token.IDENTIFIER (Lexeme = 'wireasdf')
Token.SEMICOLON (Lexeme = ';')
Token.KEYWORD (Lexeme = 'wire')
Token.IDENTIFIER (Lexeme = 'asdf')
Token.SEMICOLON (Lexeme = ';')
Token.KEYWORD (Lexeme = 'wire')
Token.IDENTIFIER (Lexeme = 'asdf')
Token.SEMICOLON (Lexeme = ';')
